tokio-util/src/time/mod.rs

```diff
@@ -12,7 +12,6 @@ use std::time::Duration;
 
 mod wheel;
 
-#[doc(inline)]
 pub mod delay_queue;
 
 pub use delay_queue::DelayQueue;
```

Comment:
```suggestion
#[doc(inline)]
pub use delay_queue::DelayQueue;
```

[Given how this is currently rendered on docs](https://docs.rs/tokio-util/0.6.7/tokio_util/time/index.html), I believe the author of this code originally wanted to use `#[doc(inline)]` on `use`, not `mod`.